src/domain/hermez/index.ts

```diff
@@ -6,26 +6,30 @@
 import * as hermezjs from "@hermeznetwork/hermezjs";
 
 export type {
-  NextForger,
   CoordinatorState,
-  DelayedWithdraw,
   Exit,
   FiatExchangeRates,
   HermezApiResourceItem,
   HermezNetworkStatus,
   HermezStatus,
-  Transaction,
-  PooledTransaction,
   HermezWallet,
+  HistoryTransaction,
   ISOStringDate,
   MerkleProof,
-  Token,
-  Withdraw,
-  SignerData,
-  SignerType,
-  JsonRpcSignerData,
+  NextForger,
+  PendingDelayedWithdraw,
+  PendingWithdraw,
+  PooledTransaction,
   RecommendedFee,
+  Token,
 } from "@hermeznetwork/hermezjs";
 
+export type { JsonRpcSignerData, SignerData } from "@hermeznetwork/hermezjs/src/signers";
```

Comment:
Why exporting the signer interfaces from here? Cannot we already do `import { Signers } from '@hermeznetwork/hermezjs'`?

Reply:
The domain is currently proxying all the types from the lib and extending some of them.

What I did for now is export only the `Signers` namespace instead of the internals of the module.

Does it make sense like this for now?

Reply:
But you should proxy all as a `Signers` instead of just reexporting all the individual interfaces right? Maybe it's better to talk about this tomorrow as I'm not understanding it properly, this typing files and modules are new to me! hahaha